test/e2e/user_workload_monitoring_test.go

```diff
@@ -381,7 +386,9 @@ func assertMetricsForMonitoringComponents(t *testing.T) {
 	} {
 		t.Run(service, func(t *testing.T) {
 			f.ThanosQuerierClient.WaitForQueryReturn(
-				t, time.Minute, fmt.Sprintf(`count(up{service="%s",namespace="openshift-user-workload-monitoring"} == 1)`, service),
+				// To avoid making the test wait for more than lookback-delta in case Prometheus
+				// wasn't able to write stale markers (because it was down), reduce the lookup period.
```

Comment:
Do we have a ticket to investigate why the stale markers are missing? The test didn't seem flaky initially so it would be good to understand what has changed and we need to fix something else.